### DISCUSSION TITLE
Enable ingredient amount by default or a a setting to it

### DISCUSSION THREAD

**Discussion by felixschndr** [mealie-recipes/mealie#3626] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When creating a new recipe the `disable ingredient amount` is always enabled.

It would be great if this was disabled by default. *In my opinion* it's always better to have quantity, unit, food and notes separated as this gives a better overview and makes scaling to different amounts of portions easier.

### Please provide a concise description of the feature that would resolve your issue.

Disable `disable ingredient amount` by default or add a setting to the env variables/settings of the app to let the user choose

### Please consider and list out some caveats or tradeoffs made in your design decision

*I* don't see any

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by felixschndr** [comment]
Edit: This is already possible in the settings of the `group`. It was set to enabled by default in release `v1.0.0` but can easily be disabled in the settings (e.g `https://<your mealie instance>/group`)

**Reply by uninvitedguest** [reply]
Can you elaborate on your comment? I'm looking to enable the ingredient amount setting and can't find it anywhere within mealie.

**Reply by felixschndr** [reply]
I am sorry, the url in the comment was broken. I fixed it. 

1. Go to your profile
2. Click on group settings
3. Toggle the last checkbox `Disable organizing recipe ingredients by units and food`
   ![image](https://github.com/mealie-recipes/mealie/assets/45742226/c55f6032-2ada-44d5-ae46-70416aaba273)

**Reply by uninvitedguest** [reply]
Appreciate it, was able to sort it out now.

**Reply by MartynasSapoka** [reply]
This has moved to household settings in the new version

**Reply by bpmartin20** [reply]
As of v3.19, I'm not finding it in either location. Apparently it moved again or went away.
